### PR TITLE
Install doc

### PR DESCRIPTION
**Pull request by Kolloom**
Env Var used in the compose.yml is INPUT_DIR and OUTPUT_DIR

Clarify the meaning of both variables and output message.
Note on -p not working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,19 +74,26 @@ python3 -m manim example_scenes.py SquareToCircle -pl
...
 ```sh
-INPUT_PATH=/path/to/dir/containing/source/code \
-OUTPUT_PATH=/path/to/dir/for/media \
+INPUT_DIR=/path/to/dir/containing/source/code \
+OUTPUT_DIR=/path/to/output/ \
 docker-compose run manim example_scenes.py SquareToCircle -l
 ```
+The command needs to be run as root if your username is not in the docker group.
+
+example_scenes.py or your own project should point to the file on the host machine.
```
Comment (eulertour) [inline README.md]: I'm not sure what you mean by "pointing" to a file here.

Comment (Kolloom) [reply README.md]: maybe `should be path to files`?

**Comment by Kolloom** [comment]
I write those documentation base on the assumption that the code is always correct.

**Comment by Kolloom** [comment]
I added a diagram for the docker setup, should be more clear

File: README.md [hunk 2]
```diff
@@ -75,18 +75,27 @@ python3 -m manim example_scenes.py SquareToCircle -pl
 Since it's a bit tricky to get all the dependencies set up just right, there is a Dockerfile and Compose file provided in this repo as well as [a premade image on Docker Hub](https://hub.docker.com/r/eulertour/manim/tags/). The Dockerfile contains instructions on how to build a manim image, while the Compose file contains instructions on how to run the image.
 
 In order to do this with the Compose file, you must set the `INPUT_PATH`
-environment variable to the directory containing your source code and the
-`OUTPUT_DIRECTORY` environment variable to the directory where you want media
-to be written.
+environment variable to the directory containing manim repository and the
```
Comment (eulertour) [inline README.md]: Manim is now stored inside the docker image, so `INPUT_PATH` should be the absolute path to the directory containing the source code.

File: README.md [hunk 3]
```diff
@@ -75,18 +75,27 @@ python3 -m manim example_scenes.py SquareToCircle -pl
...
 3. Render an animation
 ```sh
 INPUT_PATH=/path/to/dir/containing/source/code \
-OUTPUT_PATH=/path/to/dir/for/media \
+OUTPUT_PATH=/path/to/output/ \
 docker-compose run manim example_scenes.py SquareToCircle -l
 ```
+The command needs to be run as root if your username is not in the docker group.
+
+replace `example_scenes.py` with your own on the host machine for your projects.
```
Comment (eulertour) [inline README.md]: "You can replace `example_scenes.py` with any relative path from your `INPUT_PATH`"

**Comment by Kolloom** [comment]
Did the pr slip through the notification?

**Pull request merged**